[Search][Playground] support for editing search query

TattdCodeMonkey opened the pull request:
## Summary

This PR updates the Query Mode in playground to allow the user to customize the query themselves. When the query is customized the GUI for selecting fields is disabled. Changing the index setting will reset the query to the elastic generated search query.

### Screenshots
<img width="1547" alt="image" src="https://github.com/user-attachments/assets/24d94417-4e10-46dd-abaf-49c7196e17c0" />
`{query}` placeholder is required even when customized
<img width="1547" alt="image" src="https://github.com/user-attachments/assets/8522ba4b-5027-49a6-a85d-19857e76e43b" />
"Run" action is disabled when query is not valid JSON
<img width="1547" alt="image" src="https://github.com/user-attachments/assets/c741d755-4659-4dce-857f-4dfbb73b3035" />
Errors are displayed in Query Output
<img width="1547" alt="image" src="https://github.com/user-attachments/assets/34a797c9-6de6-443a-b58c-5ea8f8c5bf79" />

Query field selection is disabled when query is customized

![image](https://github.com/user-attachments/assets/0fcda6c5-a820-4470-b554-9fc1d2ed7950)


### Feature Flag

** This work is behind the `searchPlayground:searchModeEnabled` feature flag **
Enable feature with Dev Tools
```
POST kbn:/internal/kibana/settings/searchPlayground:searchModeEnabled
{"value": true}
```
OR
Enable feature in `kibana.dev.yml`
```yaml
uiSettings.overrides:
  'searchPlayground:searchModeEnabled': true
```


### Checklist

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [ ] [Documentation](https://www.elastic.co/guide/en/kibana/master/development-documentation.html) was added for features that require explanation or tutorials
- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed

TattdCodeMonkey committed:
search_playground: support for editing search query

TattdCodeMonkey commented on x-pack/solutions/search/plugins/search_playground/public/hooks/use_load_fields_by_indices.ts:
```diff
@@ -35,13 +35,16 @@ export const useLoadFieldsByIndices = ({
+  const { fields, isFetched } = useIndicesFields(selectedIndices);
 
   useEffect(() => {
-    const [queryFields, sourceFields] = getValues([
```
`getValues([])` doesn't type the return value correctly. switching to `getValues()` and destructing ensured the form fields we are using have the correct type instead of `any` or `never`.

TattdCodeMonkey committed:
search_playground: add tooltip for disabled field selection

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/styled_components_mapping'

TattdCodeMonkey committed:
search_playground: condense query placeholder error

TattdCodeMonkey commented on x-pack/solutions/search/plugins/search_playground/public/components/query_mode/query_fields_panel.tsx:
```diff
@@ -103,6 +106,29 @@ export const QueryFieldsPanel = ({
+                        'xpack.searchPlayground.viewQuery.sidePanel.fieldSelection.customized.warning.tooltip',
+                        {
+                          defaultMessage:
+                            'Field selection is not supported with a user-customized query',
```
@leemthompo copy check 🙏

TattdCodeMonkey commented on x-pack/solutions/search/plugins/search_playground/public/providers/form_provider.tsx:
```diff
@@ -75,13 +85,19 @@ export const FormProvider: React.FC<React.PropsWithChildren<FormProviderProps>>
   useEffect(() => {
     const subscription = form.watch((values) =>
-      setLocalSession(values as Partial<ChatForm>, storage)
+      setLocalSessionDebounce.run(values as PartialChatForm, storage)
```
Even before adding query editing this was spamming local storage, adding debounce slows that down considerably.

TattdCodeMonkey commented on x-pack/solutions/search/plugins/search_playground/public/utils/user_query.ts:
```diff
@@ -0,0 +1,84 @@
+  if (!userQuery.includes('{query}')) {
+    userQueryErrors = [
+      i18n.translate('xpack.searchPlayground.userQuery.errors.queryPlaceholder', {
+        defaultMessage: 'User query must contain "{query}" placeholder',
```
@leemthompo copy check 🙏

leemthompo replied:
```suggestion
        defaultMessage: 'User query must contain "{query}"',
```

TattdCodeMonkey commented on x-pack/solutions/search/plugins/search_playground/public/utils/user_query.ts:
```diff
@@ -0,0 +1,84 @@
+    userQueryErrors = [
+      i18n.translate('xpack.searchPlayground.userQuery.errors.queryPlaceholder', {
+        defaultMessage: 'User query must contain "{query}" placeholder',
+        values: { query: '{query}' },
```
i18n wants to replace the `{query}` in the default message so we give it `{query}` :/ is there a better way to handle this I'm unaware of?

TattdCodeMonkey commented on x-pack/solutions/search/plugins/search_playground/public/components/query_mode/query_viewer.tsx:
```diff
@@ -81,18 +126,39 @@ export const ElasticsearchQueryViewer = ({
+      (userElasticsearchQueryValidations?.userQueryErrors?.length ?? 0) > 0 ? (
+        <EuiSplitPanel.Inner grow={false}>
+          {userElasticsearchQueryValidations.userQueryErrors!.map((error, errorIndex) => (
+            <EuiCallOut
```
I'd rather have this displayed in the editor. And I'm investigating the ability to do that, but do far that is looking more complicated than I'd like. so this is a stop-gap until I figure it out.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/287335)
* Commit: 3915a3844bd7ab374bfb6c4d5db67bb58ddf029a

### Failed CI Steps
* [FTR Configs #66](https://buildkite.com/elastic/kibana-pull-request/builds/287335#0195d2f1-bdae-4ca3-a8d4-adf78cab3609)
* [FTR Configs #66](https://buildkite.com/elastic/kibana-pull-request/builds/287335#0195d316-9058-45c4-8e56-8aa98f69f2b7)
* [FTR Configs #75](https://buildkite.com/elastic/kibana-pull-request/builds/287335#0195d2f1-bdb6-444a-b5a4-91797737face)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287335#0195d2f1-bdb6-444a-b5a4-91797737face) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287335/jobs/0195d2f1-bdb6-444a-b5a4-91797737face/artifacts/0195d318-5a14-4ebd-bb4f-fb9f231b08ae) FTR Configs #<span></span>75 / EQL execution logic API @<span></span>ess @<span></span>serverless @<span></span>serverlessQA EQL type rules parses shard failures for EQL event query
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287335#0195d2f1-bdae-4ca3-a8d4-adf78cab3609) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287335/jobs/0195d2f1-bdae-4ca3-a8d4-adf78cab3609/artifacts/0195d316-7aa6-4190-9856-c4b217ecc3c8) FTR Configs #<span></span>66 / serverless search UI Serverless Playground Overview chat page chat works
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287335#0195d316-9058-45c4-8e56-8aa98f69f2b7) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287335/jobs/0195d316-9058-45c4-8e56-8aa98f69f2b7/artifacts/0195d32b-b4e4-41c0-88b9-cbb58fea0531) FTR Configs #<span></span>66 / serverless search UI Serverless Playground Overview chat page chat works

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287335","number":287335,"commit":"3915a3844bd7ab374bfb6c4d5db67bb58ddf029a"}],"number":287335}
buildkite-pr-comment-->

TattdCodeMonkey committed:
search_playground: remove editor schema todos

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/287472)
* Commit: 839019fe5e3f6ffc4aed3da75e88ac76624bfc99

### Failed CI Steps
* [FTR Configs #56](https://buildkite.com/elastic/kibana-pull-request/builds/287472#0195d3c7-5a20-482c-b624-1a00d9216db1)
* [FTR Configs #56](https://buildkite.com/elastic/kibana-pull-request/builds/287472#0195d3ea-374a-422a-844f-2eaa39039e2d)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287472#0195d3c7-5a20-482c-b624-1a00d9216db1) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287472/jobs/0195d3c7-5a20-482c-b624-1a00d9216db1/artifacts/0195d3ea-13a5-42bc-964c-a94964c436db) FTR Configs #<span></span>56 / serverless search UI Serverless Playground Overview chat page chat works
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287472#0195d3ea-374a-422a-844f-2eaa39039e2d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287472/jobs/0195d3ea-374a-422a-844f-2eaa39039e2d/artifacts/0195d3fd-5b0a-45cf-aa98-6a8dcf2b487d) FTR Configs #<span></span>56 / serverless search UI Serverless Playground Overview chat page chat works

### History
* :broken_heart: [Build #287335](https://buildkite.com/elastic/kibana-pull-request/builds/287335) failed 3915a3844bd7ab374bfb6c4d5db67bb58ddf029a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287472","number":287472,"commit":"839019fe5e3f6ffc4aed3da75e88ac76624bfc99"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287335","number":287335,"commit":"3915a3844bd7ab374bfb6c4d5db67bb58ddf029a"}],"number":287472}
buildkite-pr-comment-->

TattdCodeMonkey committed:
search_playground: lower session storage debounce to 100ms

TattdCodeMonkey committed:
test(search_playground): handle local session debounce latency

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/287769)
* Commit: f208fb1407a582bcfa9d3641238466156597736c

### Failed CI Steps
* [FTR Configs #80](https://buildkite.com/elastic/kibana-pull-request/builds/287769#0195d852-5c61-4a8e-9ac1-dbc0df148ca6)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/287769#0195d852-5c61-4a8e-9ac1-dbc0df148ca6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/287769/jobs/0195d852-5c61-4a8e-9ac1-dbc0df148ca6/artifacts/0195d872-adde-4dcc-8e8a-0e2e8215ec74) FTR Configs #<span></span>80 / maps app geo file upload shapefile upload should add as document layer

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/f208fb1407a582bcfa9d3641238466156597736c) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 308 | 331 | +23 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/f208fb1407a582bcfa9d3641238466156597736c) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 194.9KB | 201.3KB | +6.4KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/1d457e4b1bdc444f956f7f8feec6cc3415a4a605) | [after](https://github.com/elastic/kibana/commit/f208fb1407a582bcfa9d3641238466156597736c) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 8.0KB | 8.1KB | +117.0B |



### History
* :broken_heart: [Build #287472](https://buildkite.com/elastic/kibana-pull-request/builds/287472) failed 839019fe5e3f6ffc4aed3da75e88ac76624bfc99
* :broken_heart: [Build #287335](https://buildkite.com/elastic/kibana-pull-request/builds/287335) failed 3915a3844bd7ab374bfb6c4d5db67bb58ddf029a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287769","number":287769,"commit":"f208fb1407a582bcfa9d3641238466156597736c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287472","number":287472,"commit":"839019fe5e3f6ffc4aed3da75e88ac76624bfc99"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287335","number":287335,"commit":"3915a3844bd7ab374bfb6c4d5db67bb58ddf029a"}],"number":287769}
buildkite-pr-comment-->

joemcelroy commented:
Hey! tried this out, couple of suggestions and one bug:

- suggestion: "run" action should only be enabled when there is a query (maybe only required if {query} placeholer detected in the query pane)
- suggestion: should be an action to revert back to the elastic-generated query
- bug: the query / results split pane: the query pane scrollbar doesn't adjust for its actual size. See video 


https://github.com/user-attachments/assets/68ccba7f-bf7a-4fe9-bfd7-908e47a372d7

joemcelroy commented on x-pack/solutions/search/plugins/search_playground/public/components/query_mode/query_output.tsx:
```diff
@@ -25,10 +32,14 @@ export const ElasticsearchQueryOutput = ({
   isError,
   queryError,
+  isLoading,
 }: ElasticsearchQueryOutputProps) => {
```
one thing noticed is useQuery will retry 3 times by default. We should check if its a 400 error then return the error message without retry. 

https://github.com/user-attachments/assets/2a4b24b7-7dbc-4a4e-be46-dcdb696b8fe9

TattdCodeMonkey committed:
search_playground: disable query mode run when query is not set

TattdCodeMonkey committed:
search_playground: add reset query action when query is customized

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288133)
* Commit: ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/5a6c01ff82cbc5038df5d20f65918d6b1aba27c6) | [after](https://github.com/elastic/kibana/commit/ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 308 | 331 | +23 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/5a6c01ff82cbc5038df5d20f65918d6b1aba27c6) | [after](https://github.com/elastic/kibana/commit/ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 194.9KB | 201.8KB | +6.9KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/5a6c01ff82cbc5038df5d20f65918d6b1aba27c6) | [after](https://github.com/elastic/kibana/commit/ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 8.0KB | 8.1KB | +117.0B |



### History
* :yellow_heart: [Build #287769](https://buildkite.com/elastic/kibana-pull-request/builds/287769) was flaky f208fb1407a582bcfa9d3641238466156597736c
* :broken_heart: [Build #287472](https://buildkite.com/elastic/kibana-pull-request/builds/287472) failed 839019fe5e3f6ffc4aed3da75e88ac76624bfc99
* :broken_heart: [Build #287335](https://buildkite.com/elastic/kibana-pull-request/builds/287335) failed 3915a3844bd7ab374bfb6c4d5db67bb58ddf029a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288133","number":288133,"commit":"ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287769","number":287769,"commit":"f208fb1407a582bcfa9d3641238466156597736c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287472","number":287472,"commit":"839019fe5e3f6ffc4aed3da75e88ac76624bfc99"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287335","number":287335,"commit":"3915a3844bd7ab374bfb6c4d5db67bb58ddf029a"}],"number":288133}
buildkite-pr-comment-->

TattdCodeMonkey committed:
search_playground: fix panel sizing with resizing

TattdCodeMonkey committed:
search_playground: use null for user query empty state

undefined was causing issues where resetting the user query on indices
changes were not resulting in the query being cleared. Using null
instead resolved this issue.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288214)
* Commit: ee1e3d66aadaa1e1f209b9a6541fd0800ca59424

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/288214#0195ddfe-ab4b-47ac-9196-45f870f22c97)

### History
* :green_heart: [Build #288133](https://buildkite.com/elastic/kibana-pull-request/builds/288133) succeeded ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e
* :yellow_heart: [Build #287769](https://buildkite.com/elastic/kibana-pull-request/builds/287769) was flaky f208fb1407a582bcfa9d3641238466156597736c
* :broken_heart: [Build #287472](https://buildkite.com/elastic/kibana-pull-request/builds/287472) failed 839019fe5e3f6ffc4aed3da75e88ac76624bfc99
* :broken_heart: [Build #287335](https://buildkite.com/elastic/kibana-pull-request/builds/287335) failed 3915a3844bd7ab374bfb6c4d5db67bb58ddf029a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288214","number":288214,"commit":"ee1e3d66aadaa1e1f209b9a6541fd0800ca59424"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288133","number":288133,"commit":"ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287769","number":287769,"commit":"f208fb1407a582bcfa9d3641238466156597736c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287472","number":287472,"commit":"839019fe5e3f6ffc4aed3da75e88ac76624bfc99"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287335","number":287335,"commit":"3915a3844bd7ab374bfb6c4d5db67bb58ddf029a"}],"number":288214}
buildkite-pr-comment-->

TattdCodeMonkey committed:
search_playground: do not retry test query

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288261)
* Commit: 6ed612c9d360b71cad349782ccb861192283b99a

### Failed CI Steps
* [Jest Tests #6](https://buildkite.com/elastic/kibana-pull-request/builds/288261#0195dede-9e9c-4416-9a13-9a645242ba6c)
* [Jest Tests #6](https://buildkite.com/elastic/kibana-pull-request/builds/288261#0195df02-aae3-49f1-9c78-b89557b407ce)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288261#0195dede-9e9c-4416-9a13-9a645242ba6c) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288261/jobs/0195dede-9e9c-4416-9a13-9a645242ba6c/artifacts/0195df02-8e79-4f9d-98be-0e64214e6ec8) Jest Tests #<span></span>6 / <TableTab /> should filter the table correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288261#0195df02-aae3-49f1-9c78-b89557b407ce) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288261/jobs/0195df02-aae3-49f1-9c78-b89557b407ce/artifacts/0195df18-f557-44c9-97c6-1deab9cd330f) Jest Tests #<span></span>6 / <TableTab /> should filter the table correctly

### History
* :broken_heart: [Build #288214](https://buildkite.com/elastic/kibana-pull-request/builds/288214) failed ee1e3d66aadaa1e1f209b9a6541fd0800ca59424
* :green_heart: [Build #288133](https://buildkite.com/elastic/kibana-pull-request/builds/288133) succeeded ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e
* :yellow_heart: [Build #287769](https://buildkite.com/elastic/kibana-pull-request/builds/287769) was flaky f208fb1407a582bcfa9d3641238466156597736c
* :broken_heart: [Build #287472](https://buildkite.com/elastic/kibana-pull-request/builds/287472) failed 839019fe5e3f6ffc4aed3da75e88ac76624bfc99
* :broken_heart: [Build #287335](https://buildkite.com/elastic/kibana-pull-request/builds/287335) failed 3915a3844bd7ab374bfb6c4d5db67bb58ddf029a

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288261","number":288261,"commit":"6ed612c9d360b71cad349782ccb861192283b99a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288214","number":288214,"commit":"ee1e3d66aadaa1e1f209b9a6541fd0800ca59424"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288133","number":288133,"commit":"ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287769","number":287769,"commit":"f208fb1407a582bcfa9d3641238466156597736c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287472","number":287472,"commit":"839019fe5e3f6ffc4aed3da75e88ac76624bfc99"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287335","number":287335,"commit":"3915a3844bd7ab374bfb6c4d5db67bb58ddf029a"}],"number":288261}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288274)
* Commit: 4d5183dfd7722f1f76a27a1380aac62622d0a381

### Failed CI Steps
* [Jest Tests #17](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df73-df51-4dc8-b532-8251201badfc)
* [Jest Tests #19](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df73-df51-4d62-8c7a-7b02171bf0e8)
* [Jest Tests #19](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df98-9892-4395-9d4a-af450fd65465)
* [FTR Configs #16](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df73-df5f-453a-9e16-62f91e221477)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df73-df51-4d62-8c7a-7b02171bf0e8) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288274/jobs/0195df73-df51-4d62-8c7a-7b02171bf0e8/artifacts/0195df98-81a5-4d3d-9ec1-60a773bca855) Jest Tests #<span></span>19 / <TableTab /> should filter the table correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df98-9892-4395-9d4a-af450fd65465) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288274/jobs/0195df98-9892-4395-9d4a-af450fd65465/artifacts/0195dfad-8f79-491e-bee5-a6b1d9a12ac1) Jest Tests #<span></span>19 / <TableTab /> should filter the table correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df73-df51-4dc8-b532-8251201badfc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288274/jobs/0195df73-df51-4dc8-b532-8251201badfc/artifacts/0195df98-54f9-4689-8b4f-52b411803b6a) Jest Tests #<span></span>17 / useGetEndpointActionList hook should call the proper API
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/288274#0195df73-df5f-453a-9e16-62f91e221477) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/288274/jobs/0195df73-df5f-453a-9e16-62f91e221477/artifacts/0195df90-0859-4fae-8698-945befd97391) FTR Configs #<span></span>16 / X-Pack Accessibility Tests - Group 1 Kibana Home Accessibility Observability overview page meets a11y requirements 

### History
* :broken_heart: [Build #288261](https://buildkite.com/elastic/kibana-pull-request/builds/288261) failed 6ed612c9d360b71cad349782ccb861192283b99a
* :broken_heart: [Build #288214](https://buildkite.com/elastic/kibana-pull-request/builds/288214) failed ee1e3d66aadaa1e1f209b9a6541fd0800ca59424
* :green_heart: [Build #288133](https://buildkite.com/elastic/kibana-pull-request/builds/288133) succeeded ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e
* :yellow_heart: [Build #287769](https://buildkite.com/elastic/kibana-pull-request/builds/287769) was flaky f208fb1407a582bcfa9d3641238466156597736c
* :broken_heart: [Build #287472](https://buildkite.com/elastic/kibana-pull-request/builds/287472) failed 839019fe5e3f6ffc4aed3da75e88ac76624bfc99

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288274","number":288274,"commit":"4d5183dfd7722f1f76a27a1380aac62622d0a381"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288261","number":288261,"commit":"6ed612c9d360b71cad349782ccb861192283b99a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288214","number":288214,"commit":"ee1e3d66aadaa1e1f209b9a6541fd0800ca59424"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288133","number":288133,"commit":"ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287769","number":287769,"commit":"f208fb1407a582bcfa9d3641238466156597736c"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287472","number":287472,"commit":"839019fe5e3f6ffc4aed3da75e88ac76624bfc99"}],"number":288274}
buildkite-pr-comment-->

TattdCodeMonkey commented:
@elasticmachine merge upstream

elasticmachine commented:
There are no new commits on the base branch.

TattdCodeMonkey commented:
/ci

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/288292)
* Commit: 4d5183dfd7722f1f76a27a1380aac62622d0a381

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/5b7d5ab65ecdfae5c33dacb157331b89e83d8ec6) | [after](https://github.com/elastic/kibana/commit/4d5183dfd7722f1f76a27a1380aac62622d0a381) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 308 | 331 | +23 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/5b7d5ab65ecdfae5c33dacb157331b89e83d8ec6) | [after](https://github.com/elastic/kibana/commit/4d5183dfd7722f1f76a27a1380aac62622d0a381) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 194.9KB | 201.9KB | +7.0KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/5b7d5ab65ecdfae5c33dacb157331b89e83d8ec6) | [after](https://github.com/elastic/kibana/commit/4d5183dfd7722f1f76a27a1380aac62622d0a381) | diff |
| --- | --- | --- | --- |
| `searchPlayground` | 8.0KB | 8.1KB | +117.0B |



### History
* :broken_heart: [Build #288274](https://buildkite.com/elastic/kibana-pull-request/builds/288274) failed 4d5183dfd7722f1f76a27a1380aac62622d0a381
* :broken_heart: [Build #288261](https://buildkite.com/elastic/kibana-pull-request/builds/288261) failed 6ed612c9d360b71cad349782ccb861192283b99a
* :broken_heart: [Build #288214](https://buildkite.com/elastic/kibana-pull-request/builds/288214) failed ee1e3d66aadaa1e1f209b9a6541fd0800ca59424
* :green_heart: [Build #288133](https://buildkite.com/elastic/kibana-pull-request/builds/288133) succeeded ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e
* :yellow_heart: [Build #287769](https://buildkite.com/elastic/kibana-pull-request/builds/287769) was flaky f208fb1407a582bcfa9d3641238466156597736c
* :broken_heart: [Build #287472](https://buildkite.com/elastic/kibana-pull-request/builds/287472) failed 839019fe5e3f6ffc4aed3da75e88ac76624bfc99

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288292","number":288292,"commit":"4d5183dfd7722f1f76a27a1380aac62622d0a381"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288274","number":288274,"commit":"4d5183dfd7722f1f76a27a1380aac62622d0a381"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288261","number":288261,"commit":"6ed612c9d360b71cad349782ccb861192283b99a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288214","number":288214,"commit":"ee1e3d66aadaa1e1f209b9a6541fd0800ca59424"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/288133","number":288133,"commit":"ab84ce1d1a3a1f08c711bf1fbab488d6ef6b8e0e"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/287769","number":287769,"commit":"f208fb1407a582bcfa9d3641238466156597736c"}],"number":288292}
buildkite-pr-comment-->
